tests/conftest.py

```diff
@@ -98,7 +98,7 @@ def getAllDependencies(filePath, knownDependencies):
     ####
 
     def __init__(self):
-        self.state = tester.state()
+        self.state = tester.Chain()
```

Comment:
We should rename the `state` variable to `chain`, otherwise I suspect we'll end up with quite a bit of confusion.